bip-tap.mediawiki

```diff
@@ -398,18 +398,17 @@ derived using the first input outpoint as a sort of chain randomness to ensure
 uniqueness within the system as well as the output index and asset type to
 prevent collisions.
 The <code>asset_group_key</code> is derived as follows:
-* <code>asset_group_key</code> = <code>asset_key_internal +
-sha256(asset_key_internal || genesis_outpoint || output_index || asset_type)
-</code>
+* <code>asset_group_key = TapTweak(asset_internal_group_key, asset_group_script_root)</code>
 
 where:
-* <code>asset_key_internal</code> is a 32-byte public key as defined by
-[[../master/bip-0340.mediawiki|BIP-340]]
-* <code>genesis_outpoint</code> is the first previous input outpoint used in the
-asset genesis transaction serialized in Bitcoin wire format.
-* <code>output_index</code> (4 byte, big-endian) is the index  of the output
-which contains the unique Taproot Asset commitment in the genesis transaction. 
-* <code>asset_type</code> (1 byte) is the type of the asset being minted.
+* <code>asset_internal_group_key = SingleTweak(asset_raw_key, asset_id)</code> 
+* <code>asset_raw_key</code> is a 32-byte public key as defined by [[../master/bip-0340.mediawiki|BIP-340]]
```

Comment:
Move to be the first field defined?

Reply:
also `asset_group_raw_key=? to further delineate

Reply:
Done: https://github.com/Roasbeef/bips/pull/33/commits/7d5d456e4292c01e8c01b9f6a30d82bf633ce7c4